docs/modules/ROOT/pages/explanation/pod-security-admissions.adoc

```diff
@@ -0,0 +1,103 @@
+= Pod Security Admissions
+
+Kubernetes 1.24 removes the Pod Security Policy feature in favor of Pod Security Admission. This document explains the impact on {product} users.
+
+== Introduction
+
+Up until OpenShift 4.10, OpenShift only had Security Context Constraints (SCCs.)
```

Comment:
I assume the following is rejected by Vale, becuase to me having the period inside the parentheses looks just wrong:

```suggestion
Up until OpenShift 4.10, OpenShift only had Security Context Constraints (SCCs).
```